clustering/polus-hdbscan-clustering-plugin/plugin.json

```diff
@@ -0,0 +1,89 @@
+{
+    "name": "HDBSCAN Clustering",
+    "version": "0.4.5",
+    "title": "HDBSCAN Clustering",
+    "description": "Cluster the data using HDBSCAN.",
+    "author": "Jayapriya Nagarajan (jayapriya.nagarajan@nih.gov)",
```

Comment:
Add your name to the author section. Maybe replace Jayapriya's email with her github username.

Reply:
Done.